### ISSUE TITLE
Add sorting in `order`.

### ISSUE THREAD

**Issue by rodoabad** [benmosher/eslint-plugin-import#952]
I' like `order` to have the same the member sort rule that `sort-imports` have.

```
import {hashHistory} from 'react-router';
import {syncHistoryWithStore} from 'react-router-redux';
```

VS

```
import {syncHistoryWithStore} from 'react-router-redux';
import {hashHistory} from 'react-router'; // error
```

**Comment by jshado1** [comment]
Duplicate of #389; fix pending #629

**Comment by ljharb** [comment]
Closing as a duplicate of #389.
